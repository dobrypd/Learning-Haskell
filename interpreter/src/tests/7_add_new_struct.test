struct { int a =3; } a;
int main() {
    return;
}
