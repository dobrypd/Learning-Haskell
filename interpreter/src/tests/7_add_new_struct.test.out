Welcome to Kappa interpreter
Undefined
[(fromList [(Ident "a",fromList [(Ident "a",3)])],fromList [(Ident "main",([],[SStm (JumpS SjumpFour)]))],fromList [])]
